webapp/src/features/Dashboard/SubscriptionTable.js

```diff
@@ -324,11 +272,22 @@ const SubscriptionTable = () => {
 							variant='primary'
 							id='input-dropdown-basic'
 						>
-							<Dropdown.Item>Plan</Dropdown.Item>
-							<Dropdown.Item>E-mail</Dropdown.Item>
+							<Dropdown.Item eventKey='plan' onSelect={handleClick}>
```

Comment:
lets make it dynamic
using map

Reply:
created dashboard.constants file and added array of value and label property object.